README.md

```diff
@@ -1,3 +1,11 @@
+## Deprecated
+
+This module is no longer used, or supported, and while there are a number
+of concepts and examples worth noting in the [skeleton](/skeleton/)
+directory we do not support its use or how relevant it remains.
```

Comment:
'or how relevant it remains.'

do we need this bit at all

Reply:
https://github.com/gds-operations/gds-operations.github.io/pull/12#issuecomment-308677041 Ish.